tests/convert_response_list_to_definition.yaml

```diff
@@ -0,0 +1,16 @@
+confiigurationParameters:
```

Comment:
Is this double 'i' on purpose? I've seen it a few times throughout. If not, switch to 'configurationParameters'?